taf/plugins/pytest_sut_monitor.py

```diff
@@ -227,20 +229,23 @@ def list_rrd_folders(self):
         for dev_name in self.devices:
             collectd_folder = os.path.join(PATH_TO_RRD, dev_name)
             # List all RRD folder related to specific device
-            folders = self.exec_command('find {}/*'
-                ' -maxdepth 1 -type d -print0'.format(collectd_folder)).split('\0')
+            folders = self.exec_command(
+                'find {}/*' ' -maxdepth 1 -type d -print0'.format(collectd_folder)).split('\0')
```

Comment:
can we make this command a list instead of a string?

```
folders = self.exec_command(['find', '{}/*'.format(collectd_folder), '-maxdepth', '1', '-type', 'd', '-print0']).split('\0')
```